Calculator/calci.py

```diff
@@ -5,4 +5,7 @@ def __init__(self, a, b):
         self.b = b
 
     def add(self):
-        return self.a + self.b
\ No newline at end of file
+        return self.a + self.b
+
+    def subract(self):
+        return self.a - self.b
```

Comment:
Hey @saichandrareddy1 you need to print here instead of return

Reply:
@naveen539 I have completed my changes and testes it also please my code into the main for the deployment

Reply:
Okay Thanks :)